server/app.js

```diff
@@ -0,0 +1,47 @@
+require('dotenv').config();
+const { join } = require('path');
+const express = require('express');
+const morgan = require('morgan');
+const cookieParser = require('cookie-parser');
+const { createStream } = require('rotating-file-stream');
+const cors = require('cors');
+
+const app = express();
+
+app.use(express.json({ limit: '50mb' }));
+app.use(express.urlencoded({ extended: true, limit: '50mb' }));
+
+app.use(express.static(join(__dirname, '..', 'public')));
+app.use(cookieParser());
+
+app.set('PORT', process.env.PORT || 8080);
+
+// log all requests in the Apache combined format to the file access.log.
+// create a write stream (in append mode)
+const accessLogStream = createStream('access.log', {
+  interval: '1d',
+  size: '2M',
+  path: join(__dirname, 'log'),
+});
+
+// setup looger
+app.use(morgan('combined', { stream: accessLogStream }));
+
+// Enable CORS with various options.
+app.use(cors());
+
+// test route
+app.get('/', (req, res) => {
+  res
+    .status(200)
+    .json({ success: true, status: 200, message: 'The server is running.' });
```

Comment:
why you send the ```  success: true, status: 200 ```?  the status value will send with the response object, and no need for the boolean value.

just send the message 
``` { message: 'The server is running.' } ```